pkg/argocd/argocd.go

```diff
@@ -31,9 +32,10 @@ type (
 	}
 
 	LoginOptions struct {
-		Namespace string
-		Username  string
-		Password  string
+		Namespace  string
+		Username   string
+		Password   string
+		KubeConfig string
```

Comment:
what about kube context?

Reply:
removed most kube flags (except for timeout, kubeconfig and namespace).